sm/plugins/tedge_apt_plugin/src/main.rs

```diff
@@ -104,6 +135,35 @@ fn run(operation: PluginOp) -> Result<ExitStatus, InternalError> {
     Ok(status)
 }
 
+fn install(
+    module_name: String,
+    module_version: Option<String>,
+) -> Result<ExitStatus, InternalError> {
+    if let Some(version) = module_version {
+        run_cmd(
+            "apt-get",
+            &format!("install --quiet --yes {}={}", module_name, version),
+        )
+    } else {
+        run_cmd("apt-get", &format!("install --quiet --yes {}", module_name))
+    }
+}
+
+fn remove(
+    module_name: String,
+    module_version: Option<String>,
+) -> Result<ExitStatus, InternalError> {
+    if let Some(version) = module_version {
+        // check the version mentioned present or not
```

Comment:
This comment seems out of place here, the below lines are executing remove command.